### ISSUE TITLE
your shop is being processed.. facebook 

### ISSUE THREAD

**Issue by sfconcas** [facebookincubator/facebook-for-woocommerce#526]
Hi, 

I have installed the facebook plugin on Wordpress and after 24hrs, nothing happened. so I followed the instructions and deleted it and re installed. now it's been few hours and nothing yet. 

https://www.facebook.com/pg/JSLaserCraft

the above is my shop page 

Could you please help me?

Regards, 
Simon

**Comment by mashut** [comment]
I have the same problem: 
https://www.facebook.com/pg/arstone.eu/shop/

**Comment by sfconcas** [comment]
then we are on the same boat I guess. how long is it been for you?

**Comment by mashut** [comment]
More then 48 hours. Hope the support will be quick enough.

**Comment by sfconcas** [comment]
oh wow, then quite a long time. Did you contact anyone? I opened this ticket but I don't really know who else i should contact :( It's been over 24hrs for me

**Comment by dmitridr** [comment]
Thanks for reporting this. We see a build of of shops pending review and are investigating the cause. More updates soon.

**Comment by sfconcas** [comment]
Thank you so much, I hope you guys can find the reason and help me to get this fixed soon.

On 4 Jun 2018, at 22:50, Dmitri D <notifications@github.com<mailto:notifications@github.com>> wrote:


Thanks for reporting this. We see a build of of shops pending review and are investigating the cause. More updates soon.

**Comment by dmitridr** [comment]
@sfconcas I see you have recreated your shop a lot recently. Could you recreate it one more time then wait 48 hours or so? Each time you recreate the shop, the merchant review process will reset.

@mashut please try recreating your shop via a reset of the extension, this can be done by going to `reconfigure > advanced options > delete` and setting up again.

**Comment by dmitridr** [comment]
It can take up to 48 hours for review to complete.

**Comment by sfconcas** [comment]
Hi, 

thank you very much for taking the time to reply. I have done it already twice. so does it reset the time once you recreate the shop? I done it again this afternoon. Today Have done it twice already. Should I then wait?

Regards, 
Simon

**Comment by sfconcas** [comment]
Hi, 

I checked again today, now it's been 24hrs, anyway you guys can check on it and see what's going on. I have installed it already a few times. Is there perhaps something you can do on your side? Any help would be very much appreciated. Thank you so so much

**Comment by mashut** [comment]
I have the same situation. The reset and still processing. Please help!

**Comment by sfconcas** [comment]
@dmitridr 

Guys thank you so much, the shop is now live but all the items are out of stock, any idea on how to fix this?

Thank you so much

**Comment by mashut** [comment]
Hi everyone, 

My shop is not working all the same: https://www.facebook.com/pg/arstone.eu/shop/
Please help.

**Comment by dmitridr** [comment]
Hey all, 

Both your shops have finished review and have been approved. For the out of stock issue, see #523 
You should be able to solve it with a force resync, and if not the shops team is working on a fix that should be live within 24 hrs, if it is not already.

**Comment by dmitridr** [comment]
@sfconcas @mashut Thanks again for reporting this. Because of your report, the shops team was able to fix the issue for ~400 shops in total.

**Comment by mashut** [comment]
Thank you so much for your quick help!

**Comment by sfconcas** [comment]
@dmitridr 

Thank you so much for approving the shop. I have forced re sync a few times today and nothing happened. a few hours ago I added a few listings through wp and I saw them appearing on facebook shop and they were actually fine, ( those new listings I mean) then I run new forced re sync and the new listings disappeared, so now I am just adding the last 5 listings on my wp shop jslasercraft.co.uk I am gonna force another re sync, though for some reason doesn't want to add all the listings to facebook shop, but I will do it anyway and hope for the best. Hop you guys can give us un update. You have no idea how helpful you are, THANK YOU SO MUCH
